test/controllers/ViewTransfersControllerSpec.scala

```diff
@@ -109,17 +130,41 @@ class ViewTransfersControllerSpec extends FrontEndTestHelper {
     }
   }
 
-  def verifyConsignmentRow(viewTransfersPageAsString: String, node: Node): Unit = {
+  def checkForExpectedViewTransfersPageContent(viewTransfersPageAsString: String): Unit = {
+    viewTransfersPageAsString must include("<h1 class=\"govuk-heading-l\">View Transfers</h1>")
+    viewTransfersPageAsString must include(
+      s"""                    <th scope="col" class="govuk-table__header">Reference</th>
+          |                    <th scope="col" class="govuk-table__header">Date started</th>
+          |                    <th scope="col" class="govuk-table__header">Date transferred</th>
+          |                    <th scope="col" class="govuk-table__header">Status</th>
+          |                    <th scope="col" class="govuk-table__header">Actions</th>""".stripMargin
+    )
+    viewTransfersPageAsString must include(s"""View the history of all the consignments you have uploaded and resume incomplete or failed transfers.""")
+    viewTransfersPageAsString must include(
+      """        <a href="/homepage" role="button" draggable="false" class="govuk-button govuk-button--primary">
+        |            Back to homepage
+        |        </a>""".stripMargin
+    )
+  }
 
-    val formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy")
+  def verifyConsignmentRow(
+      viewTransfersPageAsString: String,
+      consignmentType: String,
+      node: Node,
+      index: Int,
+      statusesAndActionUrls: List[(String, String)] = allPossibleStatusesAndActionUrlsForStandardUsers
+  ): Unit = {
     val exportDate = node.exportDatetime.map(_.format(formatter)).get
     val createdDate = node.createdDatetime.map(_.format(formatter)).get
-    val status = if (node.currentStatus.`export`.contains("Completed")) "Exported" else "InProgress"
+    val (transferStatus, actionUrlWithNoConsignmentId): (String, String) =
+      if (consignmentType == "judgment") ("", "") else statusesAndActionUrls(index) // need to implement one for judgments
```

Comment:
Should be implementing the feature for judgments, and testing as well, even though the feature won't be released for judgment users just yet.